docs/sources/features/panels/graph.md

```diff
@@ -186,6 +186,14 @@ There is an option under Series overrides to draw lines as dashes. Set Dashes to
 Thresholds allow you to add arbitrary lines or sections to the graph to make it easier to see when
 the graph crosses a particular threshold.
 
+### Time Regions
+
+> Only available in Grafana v5.4 and above.
+
+{{< docs-imagebox img="/img/docs/v54/graph_time_regions.png" max-width= "800px" >}}
```

Comment:
Is this image in the grafana.org repo?

Reply:
Yes in description

> PR for docs screenshot: grafana/grafana.org#77

Reply:
Will push that image to master directly now.